src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/PackageBuilder.cs

```diff
@@ -91,7 +91,7 @@ public PackageBuilder()
         private PackageBuilder(bool includeEmptyDirectories, bool deterministic)
         {
             _includeEmptyDirectories = includeEmptyDirectories;
-            _deterministic = deterministic;
+            _deterministic = false;
```

Comment:
Can we add a comment here linking to https://github.com/NuGet/Home/issues/8601, too?

Reply:
yup, will do